databricks_cli/jobs/cli.py

```diff
@@ -87,9 +87,14 @@ def reset_cli(api_client, json_file, json, job_id, version):
         with open(json_file, 'r') as f:
             json = f.read()
     deser_json = json_loads(json)
+    # If the payload is defined using the API definition rather than the CLI
+    # extract the settings data.
+    json_settings = deser_json['new_settings'] if (
```

Comment:
Should we perhaps also take their `job_id` if provided and make it optional in the `click` annotation?

Reply:
Yeah sure, I could add that as well, seems like the right thing to do here